pkg/asset/installconfig/clustername.go

```diff
@@ -36,6 +36,12 @@ func (a *clusterName) Generate(parents asset.Parents) error {
 			return validate.ClusterName1035(ans.(string))
 		})
 	}
+	if platform.Ovirt != nil {
+		/* FIX-ME: As soon bz#1915122 get resolved remove the limitation of 14 chars for the clustername */
```

Comment:
Do not use /**/ comment blocks.
```suggestion
		// FIX-ME: As soon bz#1915122 get resolved remove the limitation of 14 chars for the clustername
```